ext/pdo/pdo_stmt.c

```diff
@@ -1873,7 +1873,12 @@ int pdo_stmt_setup_fetch_mode(INTERNAL_FUNCTION_PARAMETERS, pdo_stmt_t *stmt, in
 			mode = Z_LVAL(args[skip]);
 			flags = mode & PDO_FETCH_FLAGS;
 
-			retval = pdo_stmt_verify_mode(stmt, mode, 0);
+			/* pdo_stmt_verify_mode() returns a boolean value */
+			if (pdo_stmt_verify_mode(stmt, mode, 0)) {
+				retval = SUCCESS;
+			} else {
+				retval = FAILURE;
+			}
```

Comment:
retval = ... ? SUCCESS : FAILURE;

Reply:
That's actually smarter... I don't use ternaries very often lol